mobile/status.go

```diff
@@ -1003,7 +1003,7 @@ func GetConnectionStringForBootstrappingAnotherDevice(configJSON string) string
 //
 // Example: A mobile device (device with a camera) receiving account data from
 // a device with a screen (mobile or desktop devices)
-func InputConnectionStringForBootstrapping(cs, configJSON string) string {
+func InputConnectionStringForBootstrapping(cs string, configJSON string) string {
 	if configJSON == "" {
```

Comment:
can we don't change this, i didn't see this is a problem ..

Reply:
Well, I just thought i's a typo. Was it intended to be no-type? If so, why?

Reply:
when you see something like `cs, configJSON string` in golang, it means cs has the same type(string) as configJSON, so it's not no-type here 🙂